tests/nlu/training_data/formats/test_rasa_yaml.py

```diff
@@ -10,7 +10,6 @@
 from rasa.nlu.training_data.formats.rasa_yaml import RasaYAMLReader, RasaYAMLWriter
 
 MULTILINE_INTENT_EXAMPLES = f"""
-version: "{LATEST_TRAINING_DATA_FORMAT_VERSION}"
```

Comment:
I'd leave the version in here or add one more test example with a version, because the reader needs to be able to properly read the version, too.